plutus-playground/plutus-playground-server/plutus-playground-server.cabal

```diff
@@ -26,6 +26,7 @@ library
   exposed-modules:
       Playground.Server
       Playground.Interpreter
+      Playground.Usecases
```

Comment:
You should probably add the sources for those under `extra-source-files` (that might be causing the build to fail as well)

Reply:
this didn't work but there is http://hackage.haskell.org/package/file-embed-0.0.11/docs/Data-FileEmbed.html#v:makeRelativeToProject which seems to work

Reply:
nice